lib/services/commands/commandRegistryMongoDB.js

```diff
@@ -89,7 +89,21 @@ function updateCommand(service, subservice, deviceId, command, callback) {
 function createCommand(service, subservice, deviceId, command, callback) {
     /* eslint-disable-next-line new-cap */
     const commandObj = new Command.model();
-    const attributeList = ['name', 'type', 'value'];
+    const attributeList = [
+        'name',
+        'type',
+        'value',
+        // new Command fields
+        'execTs',
+        'status',
+        'info',
+        'onDelivered',
+        'onOk',
+        'onError',
+        'onInfo',
+        'cmdExecution',
+        'dateExpiration'
```

Comment:
Documentation should explain how IOTA behaves regarding this new fields (up to now, documentation only shows a notify example, but without any further explanation).

Reply:
IMHO these new fields are just stored, but still not used.
Only targetEntityId, targetEntityType, cmd and params are used by this and https://github.com/telefonicaid/iotagent-json/pull/800 PRs

Reply:
That's fine but I'd suggest to include that in the documentation, in this way:

* Specify the fields that are just stored
* Specify how targetEntityId, targetEntityType, cmd and params are spected to be use/behave in the case of IOTAs using the library (using IOTA-JSON example, which is *de facto* the main IOTA using the library )

Reply:
This was done in https://github.com/telefonicaid/iotagent-node-lib/pull/1560/commits/850ba02d18abd70bcf6c22598c1f675bfebddd07